tests/bearlib/aspects/aspectsDocsTest.py

```diff
@@ -0,0 +1,16 @@
+from coalib.bearlib.aspects import Root
+from coalib.bearlib.aspects.docs import Documentation
+
+
+class aspectsDocsTest:
+
+    def test_aspects_docs(self):
+
+        def check(aspect):
+            if aspect.__name__ is not 'Root':
```

Comment:
Don't check every time... It's a waste of... time ;) Better do

```python
    def test_aspects_docs(self):

        def check(aspects):
            for aspect in aspects:
                assert isinstance(aspect.docs, Documentation)
                assert aspect.docs.check_consistency()
                check(aspect.subaspects.values())

        check(Root.subaspects.values())
```

If you don't wanna check `Root`